### PR TITLE
OneMobileSDK Beta Integration Sample 

### PR DESCRIPTION
**Pull request by AndriiMoskvin**

### CODE REVIEW

File: Examples/Beta Integration/Beta Integration/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import UIKit
+import OneMobileSDK
+import PlayerControls
+
+class ViewController: UIViewController {
+    @IBAction func playVideoTouched(_ sender: Any) {
+        oneSDK
+            .then { $0.getPlayer(videoID: "577cc23d50954952cc56bc47") }
+            .dispatch(on: .main)
+            .onSuccess { [weak self] player in
```
Comment (oleksii-demedetskyi) [inline Examples/Beta Integration/Beta Integration/ViewController.swift:12]: why weak?

**Review by oleksii-demedetskyi — APPROVED** [review]

File: Examples/Beta Integration/Beta Integration/ViewController.swift (hunk 1 above)
Comment (AndriiMoskvin) [reply Examples/Beta Integration/Beta Integration/ViewController.swift:12]: Prevent `self` capturing?

**Comment by oleksii-demedetskyi** [comment]
I don't see how self can capture this future.

**Comment by AndriiMoskvin** [comment]
Gave it a clear look - it is not needed - you are right.

**Pull request merged**